src/panels/config/storage/ha-config-section-storage.ts

```diff
@@ -40,6 +43,28 @@ class HaConfigSectionStorage extends LitElement {
         .narrow=${this.narrow}
         .header=${this.hass.localize("ui.panel.config.storage.caption")}
       >
+        ${this._hostInfo
+          ? html`
+              <ha-button-menu
+                corner="BOTTOM_START"
+                slot="toolbar-icon"
+                @action=${this._moveDatadisk}
+              >
```

Comment:
Should we make it a little more prominent now that it has its own page?

Reply:
Moved to Card Actions for now